src/main/java/appeng/me/storage/CellInventory.java

```diff
@@ -98,8 +98,8 @@ private CellInventory(final ItemStack o, final ISaveProvider container) throws A
 
         this.container = container;
         this.tagCompound = Platform.openNbtData(o);
-        this.storedItems = this.tagCompound.getShort(ITEM_TYPE_TAG);
-        this.storedItemCount = this.tagCompound.getInteger(ITEM_COUNT_TAG);
+        this.storedItemTypes = this.tagCompound.getShort(ITEM_TYPE_TAG);
+        this.storedItemCount = this.tagCompound.getLong(ITEM_COUNT_TAG);
```

Comment:
What about backwards compatibility?

Reply:
`NBTPrimitive` can be loaded as any java primitive type. it doesn't vanish things when i tested

Reply:
indeed they are siletly converted, ok